drivers/remote/api/api.go

```diff
@@ -24,7 +24,8 @@ func (r *Response) GetError() string {
 // GetCapabilityResponse is the response of GetCapability request
 type GetCapabilityResponse struct {
 	Response
-	Scope string
+	Scope     string
+	Multihost bool
```

Comment:
The core changes already handle this scenario as they give precedence to the datascope.
But I am ok to make the suggested change, and transition all the checks to only look at the new `ConenctivityScope` capabiltiy.